interfaces/cython/cantera/onedim.pyx

```diff
@@ -881,6 +881,16 @@ cdef class Sim1D:
             self.sim.solve(loglevel, <cbool>refine_grid)
             return
 
+        def set_soret(soret_doms, soret):
+            for dom in soret_doms:
+                dom.soret_enabled = soret
+
+        def set_transport(gas, doms, multi):
+            gas.transport_model = multi
+            for dom in doms:
+                if isinstance(dom, _FlowBase):
+                    dom.set_transport(gas)
```

Comment:
This function doesn't need `gas` and `doms` as arguments, since they always have the values `self.gas` and `self.domains`, respectively.

Reply:
done